Source/Orts.Simulation/Simulation/RollingStocks/MSTSLocomotive.cs

```diff
@@ -239,6 +239,25 @@ public float CurrentLocomotiveSteamHeatBoilerWaterCapacityL
         //float DebugSpeed = 5.0f; // Initialise at 5 mph
         //float DebugTimer = 0.0f;
 
+        public enum SteamLocomotiveFuelTypes
```

Comment:
Lines 242-259 only apply to steam so might be better in MSTSSteamLocomotive.cs

Reply:
Ok, will do.